### ISSUE TITLE
Incorrect Powershell Code example

### ISSUE THREAD

**Issue by specul8** [MicrosoftDocs/OfficeDocs-o365seccomp#913]
In the article, you talk about the Set-MalwareFilterPolicy cmdlet but the related code area shows the Set-HostedContentFilterPolicy cmdlet being used (with the same parameters as the example below it). Looks like a copy and paste error.
-------------------------------------------
Disabling Malware ZAP To disable Malware ZAP for your O365 tenant, or a set of users, use the ZapEnabled parameter of Set-MalwareFilterPolicy, an EOP cmdlet.
In the following example, ZAP is disabled for a content filter policy named "Test".

Powershell
  Set-HostedContentFilterPolicy -Identity Test -ZapEnabled $false

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c88a1bea-b981-89e7-57db-14b7887618a8
* Version Independent ID: 16dc4701-b3f9-3d6a-f699-418d1201840c
* Content: [Zero-hour auto purge - protection against spam and malware](https://docs.microsoft.com/en-us/office365/securitycompliance/zero-hour-auto-purge#feedback)
* Content Source: [SecurityCompliance/zero-hour-auto-purge.md](https://github.com/microsoftdocs/officedocs-o365seccomp/blob/public/SecurityCompliance/zero-hour-auto-purge.md)
* Service: **o365-seccomp**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

**Comment by scanum** [comment]
@specul8  Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

@officedocsbot assign @scanum

**Comment by chrisda** [comment]
Apparently, according to the topic and the available parameters on the cmdlets:

- Malware ZAP is [Set-MalwareFilterPolicy](https://docs.microsoft.com/powershell/module/exchange/antispam-antimalware/set-malwarefilterpolicy)

- Phish/Spam ZAP is [Set-HostedContentFilterPolicy](https://docs.microsoft.com/powershell/module/exchange/antispam-antimalware/set-hostedcontentfilterpolicy).

I looked carefully and found one bad reference in the description right before the Malware ZAP procedure as described.

I took the opportunity to further clean-up the topic.

Thanks for the feedback.
